UnitsNet.Tests/CustomCode/SpecificVolumeTests.cs

```diff
@@ -41,9 +41,9 @@ public static void SpecificVolumeTimesMassEqualsVolume()
         }
 
         [Fact]
-        public static void ConstantOverSpecificVolumeEqualsDensity()
+        public static void InverseSpecificVolumeEqualsDensity()
         {
-            Density density = 5 / SpecificVolume.FromCubicMetersPerKilogram(20);
+            Density density = SpecificVolume.FromCubicMetersPerKilogram(4).Inverse();
```

Comment:
Remind me, don't we want to support `double / QuantityX` when adding support for `Inverse()`? Shouldn't we offer both?

I see the division operator is removed for several quantities, e.g. `SpecificVolume`, `CoefficientOfThermalExpansion` and maybe others.

Reply:
We don't implement `double / Quantity` for any of the other `Inverse` relationships - if we had to add an operation with `double` a more "natural looking" one would be the multiplication between inverse quantities - producing a `double` (or `Ratio`), however I don't personally think that makes a lot of sense (and even less does the `double` / `A` = `B`  version).

The two other operators that are removed are from the `Ratio` (I've simply added the `-NoInferredDivision` flag):
```
        IDivisionOperators<Ratio, ReciprocalArea, Area>,
        IDivisionOperators<Ratio, Area, ReciprocalArea>,
```
Again, this is subjective (and I'm definitely not an expert on the `InverseArea` domain 😆 )- but I don't see how anyone would expect to divide `20 %`  by `2 m^2` in order to obtain the `ReciprocalArea`..

Note that these were the only two operators defined on the `Ratio` type.

Reply:
Oh and by the way, the `Ratio` operators were only added in `v6` (I'm not 100% sure, but I think the only breaking change from v5 is the `Density`).

Reply:
Alright, I guess number divided by quantity is a bit weird. I was just concerned if we are removing something people are actual using, but they'll just have to multiply with the inverse quantity instead then.

We can try and see if any complaints come our way.